timesketch/lib/tasks.py

```diff
@@ -630,6 +634,45 @@ def run_plaso(file_path, events, timeline_name, index_name, source_type, timelin
     message = "Index timeline [{0:s}] to index [{1:s}] (source: {2:s})"
     logger.info(message.format(timeline_name, index_name, source_type))
 
+    try:
+        pinfo_path = current_app.config["PINFO_PATH"]
+    except KeyError:
+        pinfo_path = "pinfo.py"
+
+    cmd = [
+        pinfo_path,
+        "--output-format",
+        "json",
+        "--sections",
+        "events",
+        file_path,
+    ]
+
+    total_events_json = {}
+    # Run pinfo.py
+    try:
+        total_events = subprocess.run(
+            cmd, capture_output=True, check=True
+        ).stdout.decode("utf-8")
+        regex = 'parsers": (.+?})'
```

Comment:
Do we need to do a regex here? If the output from pinfo is JSON, can't we just load that up and check for the total events?

I haven't looked at the output of pinfo JSON in a long time though, so I may be missing something here :)